app/views/relay/tour.scala

```diff
@@ -84,6 +84,23 @@ object tour:
         )
       )
 
+  def subscribed(pager: Paginator[RelayTour | WithLastRound])(using PageContext) =
+    views.html.base.layout(
+      title = liveBroadcasts.txt(),
+      moreCss = cssTag("relay.index"),
+      moreJs = infiniteScrollTag
+    ):
+      main(cls := "relay-index page-menu")(
+        pageMenu("subscribed"),
+        div(cls := "page-menu__content box box-pad")(
+          boxTop:
+            h1(liveBroadcasts())
```

Comment:
Should this use `subscribedBroadcasts()`? (and the `<title>` tag above)

Reply:
Actually I had no idea those were that translation thing + I am a dirty copier. Thanks! fixed it.

Reply:
![image](https://github.com/lichess-org/lila/assets/95964955/62572807-fd34-420e-9cdf-25c9fc45c7f6)